### PR TITLE
assorted fixes for RHEL 8.3 Beta

### PR DESCRIPTION
**Pull request by teg**
A bunch of fixes and tests for RHEL 8.3 Beta. See individual patches for details.

### CODE REVIEW

**Comment by teg** [comment]
> LGTM, is this ready to be merged? It's currently set as a draft.

Just missed a test-case. I'm about to push a new version ready to be merged.

File: tools/test-case-generators/generate-test-cases [hunk 1]
```diff
@@ -81,25 +100,104 @@ def main(distro, arch, image_types, keep_image_info, store, output):
...
+                                "time.example.com"
+                            ]
+                        },
+                        "locale": {
+                            "languages": [
+                                "en_US"
+                            ],
+                            "keyboard": "dvorak"
+                        },
+#                        "firewall": {
```
Comment (ondrejbudai) [inline tools/test-case-generators/generate-test-cases:175]: why is this commented out?

Comment (teg) [reply tools/test-case-generators/generate-test-cases:175]: The firewall stage requires firewalld to be installed in the image (sad, but also fine). Sadly, we cannot currently install the `firewalld` packake on all our image types: https://github.com/osbuild/osbuild-composer/issues/806

Comment (ondrejbudai) [reply tools/test-case-generators/generate-test-cases:175]: Ah, interesting, let's add this later then.

File: test/external-repos/rhel-8.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            "rhsm": true,
+            "check_gpg": true
+        }
+    ]
+/
```
Comment (ondrejbudai) [inline test/external-repos/rhel-8.json:66]: This doesn't look like a valid JSON :stuck_out_tongue_closed_eyes:

**Comment by ondrejbudai** [comment]
Usual suspects failing the tests, otherwise we're green.

**Pull request merged**